test/test-viz.cpp

```diff
@@ -0,0 +1,311 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <tuple>
+#include <variant>
+
+#include <lorina/bench.hpp>
+
+#define SDL_MAIN_HANDLED
+
+#include <SDL.h>
+
+enum SPACING {
+    HSPACING = 2,
+    VSPACING = 2
+};
+
+enum ERROR_CODES {
```

Comment:
Type names are usually named in CamelCase or snake_case (mostly C-style). Type name describes a single entity, i.e. if type name is in plural form it means a single entity represents multiple subentities.
```suggestion
enum ErrorCode {
```